### DISCUSSION TITLE
model texture gets blurred

### DISCUSSION THREAD

**Discussion by avinashgpai** [google/model-viewer#1945] [Q&A]
### Description

I have created a earth model with additional texture in blender and the same used in here. Whenever I interact with earth model ,texture gets little blur (shaky effect) in the end of interaction. Is there anyway to stop this? I even viewed the same model in gltf-viewer site there was no blur while interacting. I appreciate some help with this. I have attached a demo site, try interact with it for blurry effect. Thank in advance 

#### Live Demo

https://earthmodel.000webhostapp.com/

**Comment by elalish** [comment]
Can you please fill out your OS, browser and perhaps make a screen recording so I know what you mean by "shaky"? We dynamically lower resolution to keep the frame rate up, then go back to full res when the model stops moving. If you want to disable this, simply set [minimumRenderScale](https://modelviewer.dev/docs/#entrydocs-loading-staticProperties-minimumRenderScale) to 1, but note this may result in low frame rate with complicated models / weak GPUs.

**Comment by avinashgpai** [comment]
Browser : chrome
OS: windows 
video link : (https://drive.google.com/drive/folders/1Uyg0bErMVgjaY42rie2aswAENy_doFG0?usp=sharing)

Sorry for the late response.

**Comment by elalish** [comment]
Yes, this is the intended behavior. I may experiment with making the transition happen a bit faster. If you really would prefer a drop in frame rate to a drop in resolution, the solution is as described above. Also keep in mind that this will happen less if your model renders faster, which you can often achieve by optimizing its textures, materials, and mesh.
